mac

```diff
@@ -56,11 +56,26 @@ else
   sudo chown -R "$(whoami)":admin /usr/local
 fi
 
+update_shell() {
+  local shell_path;
+  shell_path="$(which zsh)"
+
+  fancy_echo "Changing your shell to zsh ..."
+  if ! grep "$shell_path" /etc/shells > /dev/null 2>&1 ; then
+    fancy_echo "Adding '$shell_path' to /etc/shells"
+    sudo bash -c "echo $shell_path >> /etc/shells"
```

Comment:
If you use `sh` instead of `bash` this line will be two letters shorter and microseconds faster.